### DISCUSSION TITLE
How to change settings in the Windows executable version

### DISCUSSION THREAD

**Discussion by The-Highlander-Was-A-Documentary** [filebrowser/filebrowser#3555] [General]
When i run "filebrowser.exe", it creates "filebrowser.db".  But manually editing the file with an editor like "Kate", which is the only editor that can read it's contents, if i change the port from 8080 to 8081, which i need to do, the exe file will never start again until i delete the .db file.  The documentation for this FileBrowser project does not include how to change ports, or file locations, or ip address.  Does anyone know how to change these things on the windows version?

**Comment by mcbsys** [comment]
Just starting with this program but under Windows, type `filebrowser --help` and you will see various command line options. `filebrowser --port 8081` should change the port. There is an `address` parameter for the IP address. You could also export a config.json file (`filebrowser config export config.json`), edit it, then import (`filebrowser config import config.json`).
